jenkins/catapult/templates/release.Dockerfile

```diff
@@ -0,0 +1,29 @@
+# image name required as ARG
+ARG BUILD_IMAGE='symbolplatform/symbol-server-build-base:ubuntu-gcc-13'
+ARG RELEASE_BASE_IMAGE='symbolplatform/symbol-server-build-base:ubuntu-gcc-13'
+ARG DEBIAN_FRONTEND=noninteractive
+
+FROM ${BUILD_IMAGE} AS builder
+LABEL maintainer="Catapult Development Team"
+
+WORKDIR /tmp/build
+COPY . src/
+
+ARG COMPILER_CONFIGURATION='gcc-latest'
+ARG BUILD_CONFIGURATION='release-public'
+RUN ARCH=$([ "$(uname -m)" = "x86_64" ] && echo "amd64" || echo "arm64")  \
+	&& python3 /tmp/build/src/jenkins/catapult/runDockerBuildInnerBuild.py \
+	--compiler-configuration=/tmp/build/src/jenkins/catapult/configurations/${ARCH}/${COMPILER_CONFIGURATION}.yaml \
+	--build-configuration=/tmp/build/src/jenkins/catapult/configurations/${BUILD_CONFIGURATION}.yaml \
+	--source-path=/tmp/build/src/client/catapult \
+	--out-dir=/tmp/build/binaries
+
+FROM ${RELEASE_BASE_IMAGE}
+LABEL maintainer="Catapult Development Team"
```

Comment:
do you need two LABEL directives?

Reply:
The LABEL is only needed for the last stage since it creates the image.